Brokerages/Zerodha/ZerodhaBrokerage.cs

```diff
@@ -174,18 +174,20 @@ public void Subscribe(IEnumerable<Symbol> symbols)
             }
             foreach (var symbol in symbols)
             {
-                var market = GetSymbolMarket(symbol);
-                var instrumentToken = _symbolMapper.GetZerodhaInstrumentToken(symbol.ID.Symbol, market);
-                if (instrumentToken == 0)
+                var instrumentTokenList = _symbolMapper.GetZerodhaInstrumentTokenList(symbol.ID.Symbol);
+                if (instrumentTokenList.Count == 0)
                 {
-                    Log.Error("ZerodhaBrokerage.Subscribe(): Invalid Zerodha Instrument token");
+                    Log.Error($"ZerodhaBrokerage.Subscribe(): Invalid Zerodha Instrument token for given: {symbol.ID.Symbol}");
                     continue;
                 }
-                if (!subscribeInstrumentTokens.Contains(instrumentToken.ToStringInvariant()))
+                foreach (var instrumentToken in instrumentTokenList)
                 {
-                    subscribeInstrumentTokens.Add(instrumentToken.ToStringInvariant());
-                    unSubscribeInstrumentTokens.Remove(instrumentToken.ToStringInvariant());
-                    _subscriptionsById[instrumentToken.ToStringInvariant()] = symbol;
+                    if (!subscribeInstrumentTokens.Contains(instrumentToken.ToStringInvariant()))
+                    {
+                        subscribeInstrumentTokens.Add(instrumentToken.ToStringInvariant());
+                        unSubscribeInstrumentTokens.Remove(instrumentToken.ToStringInvariant());
+                        _subscriptionsById[instrumentToken.ToStringInvariant()] = symbol;
```

Comment:
Cool!
nit - We can perform `instrumentToken.ToStringInvariant()` once at the top and just re use it?